src/components/AvatarStack/AvatarStack.tsx

```diff
@@ -57,16 +64,16 @@ const AvatarStackComponent = React.forwardRef<HTMLUListElement, AvatarStackProps
             // to restore role manually
             // eslint-disable-next-line jsx-a11y/no-redundant-roles
             <ul className={b({'overlap-size': overlapSize}, className)} role={'list'} ref={ref}>
+                {visibleItems}
                 {hasMoreButton ? (
-                    <AvatarStackItem key="more-button">
+                    <AvatarStackItem key="more-button" style={{zIndex: 0}}>
```

Comment:
Move this object to some variable outside of component please, to avoid redundant props change

Reply:
done